src/i18n/locale-en.json

```diff
@@ -2505,6 +2506,7 @@
     "common.show_all": "Show all",
     "common.hide": "Hide",
     "common.learn_more_in_documentation": "Learn more in the GraphDB documentation",
+    "copy_to_clipboard.copied.tooltip": "IRI copied",
```

Comment:
This must be in the cluster_management section because it's used in the cluster management view